### DISCUSSION TITLE
出差外地如何用infuse添加alist？

### DISCUSSION THREAD

**Discussion by bestrooney** [AlistGo/alist#3837] [Q&A]
alist已经挂好阿里云盘，直接通过alist看到云盘内的内容，也能观看里面的电影，但是还是习惯infuse来看，主要是想在出差的时候看，但在infuse里面添加webdav，就是无法添加，地址栏中添加http://localhost:5244，用户名和密码用alist登入的，高级里面的路径和端口是不是也需要填写？或者其它还有我没有考虑到的家中和出差的什么因素？

**Comment by lianfan8937** [comment]
家里是同一局域网下的内网，出差那就是外网IP了，好像需要做一下外网穿透映射之类的吧，我也不太懂，到B站搜一下外网访问ALIST吧
